apps/nextjs-website/src/lib/fetch/types/PageSection.ts

```diff
@@ -58,13 +58,55 @@ const AccordionSectionCodec = t.strict({
   layout: t.union([t.literal('left'), t.literal('center'), t.literal('right')]),
 });
 
+const BannerLinkSectionCodec = t.strict({
+  __component: t.literal('sections.banner-link'),
+  title: t.string,
+  body: t.string,
+  theme: t.union([t.literal('light'), t.literal('dark')]),
+  reverse: t.boolean,
+  ctaButtons: t.array(
+    // TODO: Replace with CTAButtonSchema when merged
+    t.intersection([
+      t.type({
+        text: t.string,
+        href: t.string,
+        variant: t.keyof({
+          text: null,
+          outlined: null,
+          contained: null,
+        }),
+        color: t.keyof({
+          inherit: null,
+          primary: null,
+          secondary: null,
+          success: null,
+          error: null,
+          info: null,
+          warning: null,
+        }),
+      }),
+      t.partial({
+        icon: t.union([t.string, t.null]),
+        size: t.keyof({
+          small: null,
+          medium: null,
+          large: null,
+        }),
+      }),
+    ])
+  ),
```

Comment:
```suggestion
  ctaButtons: t.array(CTAButtonSchema),
```
is it possible to use the `CTAButtonSchema` instead?